File: Python developer/step07/7.03/7.3.02.py
'''

Напишите программу, которая будет эмулировать функциональность стека. Вам на вход будут поступать четыре команды:

    add <число> – добавить в стек число
    pop – удалить из стека число и напечатать его
    head – вывести текущий элемент стека
    close – завершить программу

Ввод:

Список комманд

Вывод:

Ответы на команды 

Sample Input:

add 5
add 10
head
pop
add 7
add 9
pop
pop
head
close

Sample Output:

10
10
9
7
5

'''

stack = []

while True:
    command = input().split()
    match command[0]:
        case 'add':
            stack.append(command[1])
        case 'pop':
            print(stack.pop())
        case 'head':
            print(stack[-1])
        case 'close':
            exit(0)

File: Diving into JavaScript: For Beginners/step07/7.03/7.3.02.js
/*

Напишите программу, которая запрашивает у пользователя строку, а затем выводит на экран последние два символа этой строки. 
Если строка состоит из меньше, чем 2 символов, выведите сообщение о том, что строка слишком короткая

Sample Input 1:

JavaScript

Sample Output 1:

pt

Sample Input 2:

Карта

Sample Output 2:

та

Sample Input 3:

f

Sample Output 3:

Введенная строка слишком короткая

*/

let str = prompt();
if (str.length > 2) {
console.log(str.slice(-2));
} else {
console.log('Введенная строка слишком короткая');
}
